.github/workflows/build-and-test.yml

```diff
@@ -17,38 +17,38 @@ jobs:
     strategy:
       matrix:
         include:
-          - target: aarch64-unknown-linux-gnu
-            os: ubuntu-latest
-            no-zstd-thin: true
+          # - target: aarch64-unknown-linux-gnu
+          #   os: ubuntu-latest
+          #   no-zstd-thin: true
```

Comment:
CI is passing because I temporarily disabled some targets that were giving build errors.

We need to re-enable these targets, otherwise, we'll end up dropping support for them (should we?).

Someone needs to go and uncomment these targets at`.github/workflows/build-and-test.yml`, and figure out how to make them build successfully, @freijon can you try doing that?

Reply:
I don't think I have the permission to do this?

Reply:
I tried solving this one by changing the package version that was being installed by `apt`, no success tho.